sdk/keyvault/Azure.Security.KeyVault.Keys/tests/SessionRecords/KeyResolverTests/ConstructorArgumentValidation.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Rather than do this, why not just split the test class up like we do with other classes, i.e. these are non-recorded tests in a separate class? Makes it much easier/faster in the future to add additional true unit tests.